controllers/storagecluster/cephblockpools.go

```diff
@@ -69,6 +71,22 @@ func (r *StorageClusterReconciler) newCephBlockPoolInstances(initData *ocsv1.Sto
 				},
 			},
 		},
+		{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      builtinMgrPoolName,
+				Namespace: poolNamespace,
+			},
+			Spec: cephv1.NamedBlockPoolSpec{
+				Name: ".mgr",
+				PoolSpec: cephv1.PoolSpec{
+					DeviceClass:    initData.Status.DefaultCephDeviceClass,
+					FailureDomain:  getFailureDomain(initData),
+					Replicated:     generateCephReplicatedSpec(initData, "data"),
```

Comment:
This is considered more of a metadata pool, rather than a data pool. Importantly, we don't want to set the `targetSizeRatio`, which is done inside `generateCephReplicatedSpec()` for data pools.
```suggestion
					Replicated:     generateCephReplicatedSpec(initData, "metadata"),
```